code/game/objects/items/weapons/RCD.dm

```diff
@@ -266,7 +266,7 @@
 	if(!length(possible_actions))
 		possible_actions = list()
 		for(var/action_type in subtypesof(/datum/rcd_act))
-			possible_actions += new action_type()
+			possible_actions += action_typea
```

Comment:
```suggestion
			possible_actions += action_type
```
Test it

Reply:
I did, and then I hit `a` before I committed because I was trying to move left in game